pyhole/core/plugin.py

```diff
@@ -178,6 +178,7 @@ def load_plugins(*args, **kwargs):
             __import__("pyhole.plugins", globals(), locals(), [plugin_name])
         except Exception, exc:
             LOG.error(exc)
+            raise
```

Comment:
The history only says "Better plugin exception handling." I think it's better that the bot fails when I want it to load a plugin and it fails to load it, rather than print a log message and keep going. I've gotten confused a few times when the bot looks like everything's fine, but it turns out there was a syntax error in a plugin and it didn't load.